jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/impl/admin/ProcessInstanceAdminServiceImplTest.java

```diff
@@ -424,7 +425,32 @@ public void testListSLATimer() throws Exception {
         assertNotNull(timer.getId());
         assertNotNull(timer.getTimerName());
     }
-    
+
+    @Test(timeout=10000)
+    public void testListSuspendUntilTimer() throws Exception {
+        processInstanceId = processService.startProcess(deploymentUnit.getIdentifier(), "humanTaskWithSuspendUntil");
+        assertNotNull(processInstanceId);
+
+        List<Long> ids = this.runtimeDataService.getTasksByProcessInstanceId(processInstanceId);
+        ids.forEach(id -> {
+            this.userTaskService.suspend(id, "Administrator");
+        });
+
+        Collection<TimerInstance> timers = processAdminService.getTimerInstances(processInstanceId);
+        assertNotNull(timers);
+        assertEquals(1, timers.size());
+        
+        TimerInstance timer = timers.iterator().next();
+        assertNotNull(timer.getActivationTime());
+        assertNotNull(timer.getDelay());
```

Comment:
this assertion is useless as getDelay returns long (primitive type)

```suggestion
        assertEquals(2000L, timer.getDelay());
```

Reply:
done